### PR TITLE
feat(ui_localizations): Add Norwegian (NB) localization

### PR DESCRIPTION
**Pull request by ovestoerholt**
## Description

Adds Norwegian localization.

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for _all_ changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] All unit tests pass (`melos run test:unit:all` doesn't fail).
- [ ] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is _not_ a breaking change.

[issue database]: https://github.com/firebase/FirebaseUI-Flutter/issues
[Contributor Guide]: https://github.com/firebase/FirebaseUI-Flutter/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by lesnitsky** [comment]
@ovestoerholt could you please ask someone else who speaks Norwegian to proofread this PR and approve it?

**Comment by lesnitsky** [comment]
@ovestoerholt please also update `packages/firebase_ui_localizations/lib/src/all_languages.dart` so that`nb` is correctly picked up

**Comment by ovestoerholt** [comment]
@lesnitsky Added 👍

File: packages/firebase_ui_localizations/lib/src/all_languages.dart [hunk 1]
```diff
@@ -3,6 +3,8 @@
 // BSD-style license that can be found in the LICENSE file.
 import 'dart:ui';
 
+import 'package:firebase_ui_localizations/src/lang/nb.dart';
```
Comment (lesnitsky) [inline packages/firebase_ui_localizations/lib/src/all_languages.dart:6]: please follow the same style for imports as the rest of the languages

**Review by lesnitsky — APPROVED** [review]

File: packages/firebase_ui_localizations/lib/src/all_languages.dart (hunk 1 above)
Comment (lesnitsky) [inline packages/firebase_ui_localizations/lib/src/all_languages.dart:6]: please follow the same style for imports as the rest of the languages

**Review by lesnitsky — CHANGES_REQUESTED** [review]

File: packages/firebase_ui_localizations/lib/src/all_languages.dart (hunk 1 above)
Comment (ovestoerholt) [reply packages/firebase_ui_localizations/lib/src/all_languages.dart]: Done 👍

**Review by lesnitsky — CHANGES_REQUESTED** [review]
`error - lib/src/lang/nb.dart:6:7 - Missing concrete implementations of 'getter FirebaseUILocalizationLabels.confirmDeleteAccountAlertMessage', 'getter FirebaseUILocalizationLabels.confirmDeleteAccountAlertTitle', and 'getter FirebaseUILocalizationLabels.confirmDeleteAccountButtonLabel'. Try implementing the missing methods, or make the class abstract. - non_abstract_class_inherits_abstract_member`

Seems like there are a few missing keys.

**Pull request merged**